reference/conanfile/methods.rst

```diff
@@ -651,6 +653,16 @@ When the environment variable ``CONAN_SYSREQUIRES_SUDO`` is not defined, Conan w
     - :command:`sudo` is available in the ``PATH``.
     - The platform name is ``posix`` and the UID (user id) is not ``0``
 
+Also, when the environment variable :ref:`CONAN_SYSREQUIRES_MODE <env_vars_conan_sysrequires_mode>` is not
+defined, Conan will set its value to ``enabled``. In this case, it's also possible to set the default
+value of the variable to other valid values (``verify`` or ``disabled``). This can be useful in the
+case that, for example, we want to install missing system requirements in the docker images of a CI
+server so that they are able to generate binaries (this images would run with the environment
+variable ``CONAN_SYSREQUIRES_MODE`` defined as ``enabled``), but passing ``default_mode="verify"`` as
+a parameter in the constructor of ``SystemPackageTool`` in the recipe, we will not force the
+installation of those requirements when the recipe is installed and ``CONAN_SYSREQUIRES_SUDO`` is not
+set.
```

Comment:
I don't understand this paragraph/example 😓 

If the docker images run with `CONAN_SYSREQUIRES_MODE=enabled` that value will override the `default_mode="verify"`...

Reply:
Yes, that's the idea, if you set the `CONAN_SYSREQUIRES_MODE` environment variable, that value will always be used, it takes preference from the argument in the `SystemPackageTool` constructor.
That's how you can install the requirements in the Docker images but not force the installation if you don't have that variable set. I'll try to rephrase this a little bit as it may be confusing.